script.js

```diff
@@ -403,6 +370,11 @@ function showMovieCard(movie) {
     const stars = document.querySelectorAll('.star');
     const starsOutput = document.querySelector('.star-output');
 
+    aboutMovie.classList.remove('hidden');
+    aboutMovie.classList.add('about-movie-${movie.id}');
+
+    moviePhoto.style.backgroundImage = `url(${movie.poster_url})`;
+
     movieRate.classList.remove('hidden');
```

Comment:
Where is movieRate declared?

When I run the code it says:
`TypeError: Cannot read properties of null (reading 'classList')`

This is because it can't read classList on something that is null.